src/amo/pages/UserFeedback/index.js

```diff
@@ -0,0 +1,190 @@
+  i18n: I18nType,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+|};
+
+export class UserFeedbackBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { dispatch, errorHandler, match, user } = props;
+    const { params } = match;
+
+    if (errorHandler.hasError()) {
+      log.warn('Not loading data because of an error.');
+      return;
+    }
+
+    if (!user) {
+      dispatch(
+        fetchUserAccount({
+          errorHandlerId: errorHandler.id,
+          userId: params.userId,
+        }),
+      );
+    }
+  }
+
+  onFormSubmitted: (values: FeedbackFormValues) => void = (values) => {
+    const { dispatch, errorHandler, user } = this.props;
+    const { anonymous, email, name, text, category } = values;
+
+    invariant(user, 'user is required');
+
+    dispatch(
+      sendUserAbuseReport({
+        errorHandlerId: errorHandler.id,
+        reporterEmail: anonymous ? '' : email,
+        reporterName: anonymous ? '' : name,
+        message: text,
+        reason: category,
+        userId: user.id,
+        // Only authenticate the API call when the report isn't submitted
+        // anonymously.
+        auth: anonymous === false,
+      }),
+    );
+  };
+
+  render(): React.Node {
+    const { errorHandler, user, i18n, isSubmitting, hasSubmitted } = this.props;
+
+    if (
+      errorHandler.hasError() &&
+      errorHandler.capturedError.responseStatusCode === 404
+    ) {
+      return <NotFoundPage />;
+    }
+
+    return (
+      <Page>
+        <div className="UserFeedback-page">
+          <Helmet>
+            <title>
+              {i18n.gettext('Submit feedback or report a user to Mozilla')}
+            </title>
+            <meta name="robots" content="noindex, follow" />
+          </Helmet>
+
+          <FeedbackForm
+            errorHandler={errorHandler}
+            contentHeader={
+              <Card className="UserFeedback-header">
+                <UserAvatar
+                  className="UserFeedback-header-avatar"
+                  user={user}
+                />
+
+                <h1 className="UserFeedback-header-username">
+                  {user ? user.name : <LoadingText />}
+                </h1>
+
+                <div className="UserFeedback-header-metadata">
+                  <span>{i18n.gettext('User since')}</span>
+                  {user ? (
+                    i18n.moment(user.created).format('ll')
+                  ) : (
+                    <LoadingText />
+                  )}
+                </div>
+              </Card>
+            }
+            abuseIsLoading={isSubmitting}
+            abuseSubmitted={hasSubmitted}
+            categoryHeader={i18n.gettext('Report this user to Mozilla')}
+            feedbackTitle={i18n.gettext('Send some feedback about the user')}
+            reportTitle={i18n.gettext(
+              'Report the user because they are illegal or incompliant',
+            )}
```

Comment:
I think we need to rephrase these titles, or the categories don't read correctly because they refer to a thing rather than a person (e.g. "Report **the user** ..." -> "**It** violates the law...").  My suggestion would be to say "user account" or "user profile" instead.  The other alternative is rephrasing the categories but we want to keep those consistent, ideally.

Reply:
I don't disagree and I already mentioned it to UX and Operations. No one objected so 🤷

Reply:
we tried 🤷